front/components/assistant/conversation/actions/VisualizationActionIframe.tsx

```diff
@@ -130,45 +130,41 @@ export function VisualizationActionIframe({
 }: {
   owner: WorkspaceType;
   visualization: Visualization;
-
   onRetry: () => void;
 }) {
-  const [contentHeight, setContentHeight] = useState(0);
-  const [iframeLoaded, setIframeLoaded] = useState(false);
-  const [showSpinner, setShowSpinner] = useState(true);
+  const [contentHeight, setContentHeight] = useState<number>(0);
+  const [errored, setErrored] = useState(false);
```

Comment:
If you keep a boolean, let keep the same convention:
```suggestion
  const [isErrored, setIsErrored] = useState(false);
```